src/main/scala/dpla/ingestion3/harvesters/file/DlgFileHarvester.scala

```diff
@@ -0,0 +1,154 @@
+package dpla.ingestion3.harvesters.file
+
+import java.io.{BufferedReader, File, FileInputStream, InputStreamReader}
+import java.util.zip.ZipInputStream
+
+import com.databricks.spark.avro._
+import dpla.ingestion3.confs.i3Conf
+import org.apache.commons.io.IOUtils
+import org.apache.log4j.Logger
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import org.json4s.jackson.JsonMethods._
+import org.json4s.{JValue, _}
+
+import scala.util.{Failure, Success, Try}
+
+
+/**
+  * Entry for performing a Florida file harvest
+  */
+class DlgFileHarvester(spark: SparkSession,
+                       shortName: String,
+                       conf: i3Conf,
+                       logger: Logger)
+  extends FileHarvester(spark, shortName, conf, logger) {
+
+  def mimeType: String = "application_json"
+
+  protected val extractor = new FlFileExtractor()
```

Comment:
Should there be a separate implementation of this that's for DLG, or should we generify the name of this somehow?

Reply:
just renamed for now, should be a part of the larger file harvester refactor though. Probably as a generic XmlExtractor and JsonExtractor rather than provider specific